apps/cli/templates/defaults/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "dependencies": {
+    "@webstudio-is/react-sdk": "^0.82.0",
+    "@webstudio-is/sdk-components-react-remix": "^0.82.0",
+    "@webstudio-is/sdk-components-react": "^0.82.0",
+    "@webstudio-is/form-handlers": "^0.75.0",
+    "use-sync-external-store": "^1.2.0"
```

Comment:
What use-sync-external-store for? Let's remove everywhere. I would not support react < v18